scripts/Tools/case.build

```diff
@@ -49,18 +51,22 @@ OR
     libs  = ["csmshare", "mct", "pio", "gptl"]
     allobjs = comps + libs
 
-    parser.add_argument("--clean", nargs="*", choices=allobjs,
-                        help="objects to clean"
-                        "if no arguments then clean all objects other than sharedlib objects")
+    mutex_group.add_argument("--clean", nargs="*", choices=allobjs,
+                             help="objects to clean"
+                             "if no arguments then clean all objects other than sharedlib objects")
+
+    mutex_group.add_argument("-b", "--build", nargs="+", choices=allobjs,
+                             help="libs to build")
```

Comment:
Please add documentation stating that the creation of namelists will be skipped if this argument is specified.